images-dist/modules/com.redhat.osl.bundleinstall/bundle/metadata/annotations.yaml

```diff
@@ -3,7 +3,7 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
-  operators.operatorframework.io.bundle.package.v1: logic-operator-rhel8
+  operators.operatorframework.io.bundle.package.v1: logic-operator-rhel9
   operators.operatorframework.io.bundle.channels.v1: alpha
```

Comment:
I've upgraded this already 🤔 

Maybe I missed OSL? :/

Reply:
Idk, how this should propagate during the sync. But if more needs to be done on kie-tools then I guess it is missed in midstream